docs/developer-guide/ksqldb-rest-api/ksql-endpoint.md

```diff
@@ -30,6 +30,10 @@ SELECT statements use the `/query` endpoint.
 - **streamsProperties[``property-name``]** (string): The value of the property
 - named by ``property-name``. Both the value and ``property-name`` should be
   strings.
+- **sessionVariables** (map): Optional. Starting from 0.18, the parameter `sessionVariables` accepts
+  a map of variable names and values as initial variable substitution values. See
+  [ksqlDB Variable Substitution](../../../how-to-guides/substitute-variables) for more information

```

Comment:
Does this link work without the `.md`  suffix?

Reply:
Yup, this would point to the link on the website